tests/lib/preflights/sanity_checks.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the Sanity Checker preflight."""
+
+import unittest
+
+from dftimewolf.lib import state
+from dftimewolf.lib.preflights import sanity_checks
+from dftimewolf import config
+
+
+class SanityChecks(unittest.TestCase):
+  """Tests for the Sanity Checker preflight."""
+
+  def testInitialization(self):
+    """Test initialisation of the sanity checker."""
+    test_state = state.DFTimewolfState(config.Config)
+    checker = sanity_checks.SanityChecks(test_state)
+    self.assertIsNotNone(checker)
+
+  def testInvalidDateOrder(self):
+    """Test that dates in the incorrect order produces an error."""
+    test_state = state.DFTimewolfState(config.Config)
+    checker = sanity_checks.SanityChecks(test_state)
+
+    checker.SetUp(
+        startdate='2020-10-31', enddate='2020-10-01', dateformat='%Y-%m-%d')
+    checker.Process()
+
+    # The incorrect date order should mean an error was encountered
+    self.assertEqual(
```

Comment:
For each of these tests, can we also check that the error message is the one we expect?

Reply:
Done.